tests/src/test/java/alluxio/client/fs/io/FileOutStreamIntegrationTest.java

```diff
@@ -247,4 +248,31 @@ public void cancelWrite() throws Exception {
       Assert.assertEquals(0, worker.getUsedBytes());
     }
   }
+
+  @Test
+  public void getStatusBeforeClose() throws Exception {
```

Comment:
Before this change, this integration test does not pass for `CACHE_THROUGH`. The stream will fail at `close()`.